apis/metal3.io/v1alpha1/baremetalhost_validation_test.go

```diff
@@ -793,8 +793,9 @@ func TestValidateCreate(t *testing.T) {
 	}
 
 	for _, tt := range tests {
+		webhook := &BareMetalHost{}
 		t.Run(tt.name, func(t *testing.T) {
-			if err := tt.newBMH.validateHost(); !errorArrContains(err, tt.wantedErr) {
+			if err := webhook.validateHost(tt.newBMH); !errorArrContains(err, tt.wantedErr) {
```

Comment:
We don’t need to create a new webhook object in the apis folder, we can keep it as is. Just make the necessary changes in the internal folder.